metadata.json

```diff
@@ -35,23 +33,20 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "6",
         "7",
         "8.1"
```

Comment:
Generally speaking we don't list minor versions. Perhaps this should be changed too?

Reply:
I had pushed a commit that removed the minor version but I removed it as I don't know if there was a particular reason to listed the minor version. I've fished it out of the reflog.